### PR TITLE
merge 2.3 -> master

### PR DESCRIPTION
**Pull request by momoss**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,6 +50,7 @@ npm-debug.log
 !/docker/akeneo-behat.conf
 !/docker/httpd.conf
 .env
+docker-compose.yml
```
Comment (damien-carcel) [inline .gitignore]: no

File: src/Akeneo/Platform/CommunityVersion.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
 class CommunityVersion
 {
     /** @staticvar string */
-    const VERSION = '2.3.17';
+    const VERSION = '2.3.18';
```
Comment (damien-carcel) [inline src/Akeneo/Platform/CommunityVersion.php]: 3.0.0

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php [hunk 3]
```diff
@@ -24,6 +27,15 @@ class FileValidator extends ConstraintValidator
         BaseFileValidator::MIB_BYTES => 'MiB',
     ];
 
+    /**
+     * @TODO Remove null parameter in merge master
```
Comment (Doodoune) [inline src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: Don't forget that ;)

File: src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php [hunk 4]
```diff
@@ -133,4 +144,54 @@ protected static function moreDecimalsThan($double, $numberOfDecimals)
...
+
+        $mappedMimeTypes = $this->extensionToMimeTypeMapping[$this->getExtension($fileInfo)];
+
+        try {
+            $mimeType = null !== $fileInfo->getUploadedFile() ?
+                $fileInfo->getUploadedFile()->getMimeType() :
+                $fileInfo->getMimeType();
+        } catch (\LogicException $e) {
+            // TODO Add requirement for fileinfo extension on merge master
+            // TODO Remove this try-catch on merge master
```
Comment (Doodoune) [inline src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/FileValidator.php]: Don't forget that ;)

**Comment by Doodoune** [comment]
Some people merge on master so we need to do another merge :
https://github.com/akeneo/pim-community-dev/pull/9235

I close this PR

**Pull request closed without merging**